### ISSUE TITLE
[Bug] <Duckduckgo's Tracking Protection>

### ISSUE THREAD

**Issue by ImpenetrableNoble** [duckduckgo/Android#3096]
### Describe the bug

I'm experiencing tracking protection errors where whenever I try to disable the VPN the key icon that appears near my save battery mute volume and WiFi system wouldn't disappear by itself.

### How to Reproduce

Open duckduckgo

Double check tracking protection feature for any errors.

### Expected behavior

Improve tracking protection feature without disturbing client connections that may bother YouTube, discord or other social media apps that interacts with duckduckgo.

### Environment

```markdown
- DDG App Version: 5.156.6
- Device: SM-A215U
- OS:One UI Core Version 4.1
```

**Comment by ImpenetrableNoble** [comment]
@aitorvs Yes I had always-on setting enabled, however I wanted to disable all of my social media applications that contain duck duckgo's VPN protection but apparently it wouldn't allow me to do so due to duckduckgo's cilent connection that's causing errors against my social media apps I use whenever I try re-enable tracking protection.

**Comment by aitorvs** [comment]
When always on is enabled the key will stay, even if AppTP is disabled. This is an Android behavior, nothing we can really do about it.

I am confused about the second part, ie. issues with social media apps. Does unprotecting those apps not fix the issues?
Unprotected apps bypass AppTP entirely and that's why eg. browsers are unprotected by default.
